src/main/java/bitbot/Application.java

```diff
@@ -0,0 +1,162 @@
+package bitbot;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Application {
+    private static final String PATH_TO_FILE = "./data/Bitbot.txt";
```

Comment:
I like your use of screaming snake case for final constants like relative filepath.